pipeline-modules/common/gcp/pipeline-config.yaml

```diff
@@ -33,11 +34,13 @@ inputs:
       description: Cloud Storage bucket for logs generated by the pipeline.
       type: string
       default: ""
+      format: regex
     pipeline_machine_type:
       title: Machine Type
       description: Type of the VM used to run the pipeline.
       type: string
       default: UNSPECIFIED
+      format: regex
```

Comment:
What exactly does this do?

Reply:
i'm not sure - when I reconciled the pipeline-config.yaml files across app-service & deployment-service, this was one diff so I took it.

Reply:
It looks like it is a carry-over from the infra-service but I have no idea what it does over there... @petmal

Reply:
The format just defines that the string value must be _"A regular expression, which should be valid according to the [ECMA 262](https://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf) dialect."_
A bit strange for say machine_type that I believe is just an enum.

Reply:
Yes. It is originally from here: https://github.com/cldcvr/vanguard-api/commit/322325eff84ee21d937d6582559fd075f89ccd17
At that time we used regexp to validate the values.
i think that the current format attribute usage is incorrect. It does not validat the value against a given regexp but rather validates the value is a regexp.

Reply:
See: https://json-schema.org/understanding-json-schema/reference/string.html#built-in-formats

Reply:
Well i think it is out of scope of this PR to remove it as it is also in the infra-service. We should file another jira to get rid of it if it isn't needed anymore (or fix the validation if it is still needed)